added transferring action repositories section (#28973)
github/docs: content/actions/creating-actions/publishing-actions-in-github-marketplace.md

Reviewer: branch

1. Select **Publish this action to the {% data variables.product.prodname_marketplace %}** to remove the check from the box.
1. Click **Update release** at the bottom of the page.

## Transferring action repositories

Action repositories can be moved by exporting or cloning the original repository, making a new repository on the target platform, and then pushing the code to the new repository.

> - A repo can be transferred and actions will continue to work because there is a redirect in place (users of the action won't have to update their workflows)

Reviewer 2: ```suggestion
When a repository admin transfers an action repository, {% data variables.product.prodname_dotcom %} automatically creates a redirect from the previous URL to the new URL, meaning workflows that use the affected action do not need to be updated.
```

I made some edits here to use active voice and adhere to our [style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md#terms-to-use-or-avoid).

> - An action in the marketplace is linked to a repository by ID, allowing for new releases to be published under the same marketplace listing from the "new" repo
> - If an action repository is deleted, so is the marketplace listing. The unique `name` identifier becomes available again.

Reviewer 2: ```suggestion

Actions published on {% data variables.product.prodname_marketplace %} are linked to a repository by their unique `name` identifier, meaning you can publish new releases of an action from the transferred repository under the same {% data variables.product.prodname_marketplace %} listing. If an action repository is deleted, the {% data variables.product.prodname_marketplace %} listing is also deleted, and the unique `name` identifier becomes available.
```

A couple more changes here to meet our [style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md) and add some extra clarity:

- We use variables for product names; for example, `{% data variables.product.prodname_marketplace %}` for "GitHub Marketplace". You can find our variables in the [`data/variables`](https://github.com/github/docs/tree/main/data/variables) folder 🙂
- I combined the two bullets since they are closely related, and I fleshed out the sentences to provide readers with more context.

> - The `verified` status seen on GitHub org profiles is different from the marketplace `verified` badge. By transferring an action repo, the owner changes and the marketplace badge will be lost unless the new owner is a verified publisher too.

Reviewer 2: ```suggestion

{% note %}

**Note:** The "Verified" badge seen on an organization's {% data variables.product.prodname_dotcom %} profile is different from the verified creator badge on {% data variables.product.prodname_marketplace %}. If you transfer an action repository, the {% data variables.product.prodname_marketplace %} listing will lose the verified creator badge unless the new owner is also a verified creator.

{% endnote %}
```

Since this is an important callout for someone thinking about transferring an action repository, I've put this in a note block so it will appear as a bright blue callout on the page (for reference, see "[Publishing an action](https://docs.github.com/en/actions/creating-actions/publishing-actions-in-github-marketplace#publishing-an-action)" in this article). I've also made some style guide updates to use variables and active voice.

1. Select **Publish this action to the {% data variables.product.prodname_marketplace %}** to remove the check from the box.
1. Click **Update release** at the bottom of the page.

## Transferring an action repository

> - You can transfer an action repository to another user or organization. For more information, see "[AUTOTITLE](/repositories/creating-and-managing-repositories/transferring-a-repository)."
> 
> - When a repository admin transfers an action repository, {% data variables.product.prodname_dotcom %} automatically creates a redirect from the previous URL to the new URL, meaning workflows that use the affected action do not need to be updated.
> 
> - Actions published on {% data variables.product.prodname_marketplace %} are linked to a repository by their unique `name` identifier, meaning you can publish new releases of an action from the transferred repository under the same [{% data variables.product.prodname_marketplace %}] listing. If an action repository is deleted, the {% data variables.product.prodname_marketplace %} listing is also deleted, and the unique `name` identifier becomes available.

Reviewer 2: ```suggestion
You can transfer an action repository to another user or organization. For more information, see "[AUTOTITLE](/repositories/creating-and-managing-repositories/transferring-a-repository)."

When a repository admin transfers an action repository, {% data variables.product.prodname_dotcom %} automatically creates a redirect from the previous URL to the new URL, meaning workflows that use the affected action do not need to be updated.

Actions published on {% data variables.product.prodname_marketplace %} are linked to a repository by their unique `name` identifier, meaning you can publish new releases of an action from the transferred repository under the same {% data variables.product.prodname_marketplace %} listing. If an action repository is deleted, the {% data variables.product.prodname_marketplace %} listing is also deleted, and the unique `name` identifier becomes available.
```

We can delete the bullets here—since these are distinct pieces of information rather than (for example) a list of options or substeps related to a specific action, this makes more sense as separate paragraphs 🙂

I also removed the brackets around one of the instances of `{% data variables.product.prodname_marketplace %}`. We only need the curly braces for the variable to render properly, and any other set of parentheses will appear in the docs themselves.

> **Note:** The "Verified" badge seen on an organization's {% data variables.product.prodname_dotcom %} profile is different from the verified creator badge on {% data variables.product.prodname_marketplace %}. If you transfer an action repository, the {% data variables.product.prodname_marketplace %} listing will lose the verified creator badge unless the new owner is also a verified creator.

Reviewer 2: ```suggestion
{% note %}

**Note:** The "Verified" badge seen on an organization's {% data variables.product.prodname_dotcom %} profile is different from the verified creator badge on {% data variables.product.prodname_marketplace %}. If you transfer an action repository, the {% data variables.product.prodname_marketplace %} listing will lose the verified creator badge unless the new owner is also a verified creator.

{% endnote %}
```

One more small change here—for the note to render properly as text inside a blue callout, we need to include the `{% note %}` and `{% endnote %}` tags.
